content/zh-cn/nodes.md

```diff
@@ -0,0 +1,17 @@
+---
+title: 节点
+status: Completed
+category: 概念
+---
+
+## 是什么
+
+节点是一台能与其他计算机（或节点）协同工作以完成一个共同任务的计算机。以你的笔记本电脑、调制解调器或打印机为例。它们都通过你的 wifi 网络进行通信和协作，各自代表一个节点。在[云计算](/zh-cn/cloud_computing/)中，节点可以是一台物理计算机，也可以是一台虚拟计算机（即虚拟机），甚至可以是[容器](/zh-cn/container/)。
+
+## 解决的问题
+
+虽然一个应用程序可以（很多应用程序确实是）运行在一台独立的机器上，但这样做存在一些风险。也就是说，底层系统的故障将会破坏应用程序。为了解决这个问题，开发人员开始创建[分布式应用程序](/zh-cn/distributed_apps/)，其中每个进程都在自己的节点上运行。因此，节点作为[集群](/zh-cn/cluster/)或节点组的一部分运行着应用程序或进程，而这些节点一起工作以实现共同的目标。
```

Comment:
Recently, `distributed_apps.md` is renamed to `distributed-apps.md`. 😊
- https://github.com/cncf/glossary/blob/dev-zh/content/zh-cn/distributed-apps.md
- https://glossary.cncf.io/zh-cn/distributed-apps/
```suggestion
虽然一个应用程序可以（很多应用程序确实是）运行在一台独立的机器上，但这样做存在一些风险。也就是说，底层系统的故障将会破坏应用程序。为了解决这个问题，开发人员开始创建[分布式应用程序](/zh-cn/distributed-apps/)，其中每个进程都在自己的节点上运行。因此，节点作为[集群](/zh-cn/cluster/)或节点组的一部分运行着应用程序或进程，而这些节点一起工作以实现共同的目标。
```